src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,335 @@
+
+A role granted directly on a cluster is unchanged if the cluster is moved into or out of a folder.
+
+The following roles allow management of folders and their contents:
+
+- **Folder Admins** can fully manage the contents of a folder, including the ability to create, rename, and move, or delete its descendent folders and clusters and to manage access to them. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendent folders.
+
+    A user with the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator](authorization.html#org-administrator) role can grant themselves, another user, or a service account the Folder Admin role.
+
+    To create a cluster in a folder, the user or service account must also have the Cluster Administrator or [Cluster Creator](authorization.html#cluster-creator) role on that folder. To delete a cluster, the user or service account must have the Cluster Administrator role on the cluster directly or by inheritance.
+
+- **Folder Movers** can rename or move a folder's contents, but cannot create or delete folders or clusters, and cannot manage access to them. The Folder Mover role grants the ability to move clusters within the folder even if the user or service account does not have direct access to the cluster.
+
+    A user with the Cluster Administrator or the Folder Admin role can grant another user or a service account the Folder Mover role.
+
+The remainder of this page shows how to create folders, manage access to them, and use them to organize your clusters.
+
+## Initial setup
+
+### Gather required information
+
+Your user account must have the following roles:
+
+- [Org Administrator](authorization.html#org-administrator)
+
+1. Sign in to the {{ site.data.products.db }} Console.
+
+1. From the **Organization** **Information** page, make a note of your organization's ID.
+
+1. [Create a service account](managing-access.html#create-a-service-account) and grant it the following roles at the organization level: <ul><li>Organization Admin</li><li>Cluster Admin</li>
+
+    When prompted, create an API key for the service account and make a note of it.
+
+### Grant the `FOLDER_ADMIN` role to the service account
+
+Folders inherit roles granted at the organization scope, and descendent folders inherit roles granted higher in the hierarchy.
+
+To create folders and manage access to them, your service account must have the `FOLDER_ADMIN` role at the level of the organization.
+
+1. To grant the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+            --url 'https://management-staging.crdb.io/api/v1/roles/<user-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+            --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace:
```

Comment:
@nickvigilante I had to remove the blank line on line 97 to avoid the sub-list breaking the numbered list.